### ISSUE TITLE
Attractor Obelisk won't work with Endermen correctly (?)

### ISSUE THREAD

**Issue by notchriss** [SleepyTrousers/EnderIO#3135]
Endermen just stand around and stare at it instead of approaching. It does seem to work in a way, i.e. they don't move around, they don't go away, they just stand and stare, so there must be some effect. But every other mob I tested with (skeletons, spiders, blazes, etc) rushes towards attractor while Endermen do not.

I am aware a similar issue was posted before and closed as fixed but it does not seem fixed to me :( Using the latest beta build EnderIO-1.7.10-2.3.0.422_beta.jar, but I had this issue when I was using latest stable build (EnderIO-1.7.10-2.2.8.381.jar) as well.

**Comment by wlhlm** [comment]
Works perfectly fine for me (with EIO 422):
![2016-01-16_10 42 01](https://cloud.githubusercontent.com/assets/1928113/12371746/8fd5f9e6-bc3e-11e5-8b5f-10c5515e4cb3.png)

Though, I've noticed a bug: The attraktor effect will remain even after the Obelisk gets destroyed by the Ender Dragon (hard to see in the following screenshot).
![2016-01-16_10 43 21](https://cloud.githubusercontent.com/assets/1928113/12371750/dac2fc4c-bc3e-11e5-94a9-0cb32e6407a6.png)

**Comment by notchriss** [comment]
http://postimg.org/image/yx3ezlw1x/

Maybe you can spot something wrong in my screenshot then. The top of the attractor is covered with a block, but it doesnt stop it from working when it comes to other kinds of mobs. 

Note that while you can  see a MinefactoryReloaded Autospawner in the screenshot, that's not where the endermen come from. They spawn from a regular endermen spawner. Not sure if it has any importance. 

And ofcourse attractor is powered and has soul vial with enderman inside. You can see some endermen standing in the screenshot, that's the closest they get to the attractor, and then keep standing there. I am not sure if they actually walk or just stay where they spawn.

**Comment by wlhlm** [comment]
Did you make sure the Endermen are actually in the AOE of the Obelisk? There's a button in the GUI which shows the range.

**Comment by notchriss** [comment]
Yes they are.

**Comment by tinproject** [comment]
I can confirm this issue in Direwolft20 1.10 pack, Enderio version 2.3.0.417 (according to changelog), Endermen won't notice the obelisk. 
But I notice that Endermen would not be attracted if a capacitor is used in the obleisk, but they will be attracted if there is no capacitor inside.

**Comment by tinproject** [comment]
I feel this is related to: #1668 and #1131

**Comment by notchriss** [comment]
It seems capacitor may be somehow related. After I removed the octadic one, one enderman immediately rushed to the attractor. But that was the last one, the rest of them, and the newly spawned ones, just keep standing and staring as usual. Since then I've tried to put capacitor back in, take it out again, switch to double-layered capacitor, take the vial out and put it back in, nothing helped

**Comment by yepidoodles** [comment]
My server just crashed (Using the latest version of EIO) after an Enderman Walked in range of the Obelisk

https://www.dropbox.com/s/duyfk6xrydrtx85/crash-2016-01-22_20.26.38-server.txt?dl=0

**Comment by HenryLoenwind** [comment]
You are using an outdated version of KCauldron that crashes with FakePlayers. That's unrelated to this issue.

**Comment by ShadowDrakken** [comment]
It's not working on wither skeletons (and possibly other mobs) spawned from an Powered Spawner either it seems.

Forge 12.17.0.1987
EnderIO 3.0.0.46 (latestest as of this posting)

No mob AI modifications, plenty of RF, proper soul jars configured...

No crashes or events to report, it just doesn't do anything except eat RF and look pretty.

**Comment by AlienXtream** [comment]
i setup some cursed earth in the end with the attractor and.... nothing. im running EIO 422. im sad now :(. and before you ask, yes, it IS setup correctly

**Comment by FinalDoom** [comment]
This is still an issue in 428.

I've got two powered spawners near a hole with an attractor over it. One spawner is endermen, one is blazes. I've got the obelisk on a timer so it's on for 5 seconds off for 20, to give them time to fall down the hole.

Blazes immediately surround the obelisk, then fall when it turns off.

Endermen freeze in their tracks and don't move. They don't do anything different with the attractor on constantly or any range all the way down to on for just a half second. The first enderman of four or 5 I hit jumped in the hole after I hit him. No other enderman jumped in the hole. Several chased me until I led them in the hole or killed them, one got mad at me for a split second then stopped in his tracks again, I think because the obelisk turned back on.

Has this been fixed in some version on in the FTB packs yet?
